framework/src/org/checkerframework/common/value/ReflectiveEvaluator.java

```diff
@@ -288,7 +288,7 @@ public Object evaluateStaticFieldAccess(
             for (Object[] arguments : listOfArguments) {
                 try {
                     results.add(constructor.newInstance(arguments));
-                } catch (ReflectiveOperationException e) {
+                } catch (Exception e) {
```

Comment:
This try/catch is nested within another try/catch for `ReflectiveOperationException`.
Should the catch type of the outer try/catch also be changed to `Exception`?
Can the scope of the outer try/catch be limited to whichever other method might throw this exception?